ui/src/components/applications/Applications.vue

```diff
@@ -63,7 +63,7 @@
             {{props.row.external ? $t('applications.open') : $t('applications.settings')}}
           </a>
           <div
-            v-if="props.row.editable == 1 && !props.row.legacy && view.isRoot"
+            v-if="props.row.editable == 1 && !props.row.legacy && hideUninstall === 'disabled' && (view.isRoot || ( view.isAdmin && props.row.id !== 'nethserver-httpd'))"
```

Comment:
In this case, if `hideUninstall ` is `enabled` we hide the kebab totally also for root, then pins and shortcuts will be hidden. We can change it to:
```html
v-if="props.row.editable == 1 && !props.row.legacy && (view.isRoot || (view.isAdmin && hideUninstall === 'disabled'))"
```

Reply:
done